src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.ServiceModel.Duplex.csproj">
-      <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net45;win8;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Thanks, based on your additional details we do want to 

> retarget the ref and make sure you harvest both the old ref and old lib that didn't have the API changes.

For the first part of that in my local repo I just updated PackageTargetFramework and NuGetTargetMoniker like so...
```
<PackageTargetFramework>netstandard2.0</PackageTargetFramework>
<NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
```
And I updated the associated project.json to...
```
{
  "frameworks": {
    "netstandard2.0": {
      "dependencies": {
        "NETStandard.Library": "2.0.0-preview1-25131-01",
        "System.Security.Principal.Windows": "4.4.0-beta-24911-02",
        "System.ServiceModel.Primitives": "4.4.0-beta-25221-02",
        "System.Reflection.DispatchProxy": "4.4.0-beta-24911-02",
        "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24911-02"
      }
    }
  }
}
```
Does that look correct?